docs/advanced/api_examples.yml

```diff
@@ -13,7 +13,7 @@
 #     cluster.  Use any integer suffix for N starting with 0.  For example,
 #     NODE_3 is replaced with the hostname for a hypothetical fourth cluster
 #     node.  (Only NODE_0 is implemented.  See FLOC-1288.)
-
+# Add a NODE_1 / implement FLOC-1288 first.
```

Comment:
@itamarst Reports that NODE_1 already exists and closed that issue. Remove this and the comment above.